server/libs/infra/src/infra.config.ts

```diff
@@ -26,17 +29,24 @@ function parseRedisConfig(): RedisOptions {
 
 export const redisConfig: RedisOptions = parseRedisConfig();
 
-export const bullConfig: BullModuleOptions = {
+interface CustomBullModuleOptions extends BullModuleOptions {
+  onError?: (error: any) => void;
```

Comment:
If there is no type here, how did you figure it this gets called anywhere?

Reply:
It seems the guidance on stackoverflow wasn't as great. :)